extensions/react-widget/src/components/DocsGPTWidget.tsx

```diff
@@ -265,7 +295,7 @@ const DotAnimation = styled.div`
 const Delay = styled(DotAnimation) <{ delay: number }>`
   animation-delay: ${props => props.delay + 'ms'};
 `;
-const PromptContainer = styled.form<{ size: string }>`
```

Comment:
@ayaan-qadri I was thinking if you could just replace the need of passing the sizes to different styled components.
Having an object of dimensions for small, medium, large and the custom.
So that the width, height could be controlled from the parent directly.

Reply:
Now we can control dimensions from sizesConfig.